notify-host.shにjqコマンドの存在確認を追加

## 問題点

`.claude/hooks/notify-host.sh` がjqコマンドを使用していますが、jqがインストールされていない環境での事前チェックがありません。そのため、jqが利用できない環境で実行するとエラーで失敗します。

## 改善提案

`install-gh.sh` と同様に、jqコマンドの存在確認とエラーメッセージを追加することを提案します。

### 実装例

```bash
#!/bin/bash
if [ -z "$WORKSPACE_DIR" ]; then
  echo "[ERROR] 環境変数 'WORKSPACE_DIR' が設定されていません。"
  exit 1
fi

if ! command -v jq >/dev/null 2>&1; then
  echo "[ERROR] jq が見つかりません。jq をインストールしてください。"
  exit 1
fi

jq -n --arg msg "${1:-Done}" --arg title "${2:-Dev Container}" \
  '{message: $msg, title: $title}' > "${WORKSPACE_DIR}/.devcontainer/host-notifier.json"
```

## 期待効果

- jqが利用できない環境でも、明確なエラーメッセージが表示される
- install-gh.shと同様の堅牢な実装パターンに統一される
- デバッグが容易になる

---
🤖 Generated with [Claude Code](https://claude.ai/claude-code)

--- a/.claude/hooks/notify-host.sh
+++ b/.claude/hooks/notify-host.sh
@@ -4,5 +4,10 @@ if [ -z "$WORKSPACE_DIR" ]; then
   exit 1
 fi
 
+if ! command -v jq &> /dev/null; then
+  echo "[ERROR] jq が見つかりません。jq をインストールしてください。"
+  exit 1
+fi
+
 jq -n --arg msg "${1:-Done}" --arg title "${2:-Dev Container}" \
   '{message: $msg, title: $title}' > "${WORKSPACE_DIR}/.devcontainer/host-notifier.json"

